.github/workflows/style.yml

```diff
@@ -0,0 +1,26 @@
+name: Style Check
+on: [push]
+
+jobs:
+    build:
+        name: Run Style check
+        runs-on: ubuntu-latest
+        strategy:
+          fail-fast: false
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Build Docker image
+              run: docker compose build linter
+
+            - name: Make .env stub
+              run: touch .env
+
+            - name: Check todo_app/
+              run: docker compose run linter poetry run pycodestyle --max-line-length=200 todo_app
```

Comment:
It feels slightly off that we're using the same docker compose service for both linting as in _checking_ the style, and _fixing_ the style which leads to these slightly ugly commands. I've suggested a possible alternative in the Docker-compose file